sklearn/covariance/graph_lasso_.py

```diff
@@ -79,8 +79,9 @@ def alpha_max(emp_cov):
 # The g-lasso algorithm
 
 def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
-                max_iter=100, verbose=False, return_costs=False,
-                eps=np.finfo(np.float).eps, return_n_iter=False):
+                enet_tol=1e-4, max_iter=100, verbose=False,
+                return_costs=False, eps=np.finfo(np.float).eps,
```

Comment:
That wasn't something I changed but I agree; I made the switch here and also in one other place (line 594).